### ISSUE TITLE
Rule proposal: AX_ARIA_03

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#12]
I think that this would make a really good linter rule. https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_03

Elements with ARIA roles must have all required attributes for that role

Good:

```html
<!-- Good: the checkbox role requires the aria-checked state -->
<span role="checkbox" aria-checked="false" aria-labelledby="foo" tabindex="0"></span>
```

Bad:

```html
<!-- Bad: the checkbox role requires the aria-checked state -->
<span role="checkbox" aria-labelledby="foo" tabindex="0"></span>
```

The [WAI-ARIA spec](http://www.w3.org/TR/wai-aria/roles#requiredState) has more information on "Required States and Properties"

**Comment by lencioni** [comment]
I think AX_ARIA_10 is very similar and would also make a good linter rule: https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_10
